client/src/pages/admin/MergeLocations.js

```diff
@@ -0,0 +1,509 @@
+import { Button, Callout } from "@blueprintjs/core"
+import styled from "@emotion/styled"
+import { DEFAULT_SEARCH_PROPS, PAGE_PROPS_NO_NAV } from "actions"
+import API from "api"
+import { gql } from "apollo-boost"
+import { LocationOverlayRow } from "components/advancedSelectWidget/AdvancedSelectOverlayRow"
+import AdvancedSingleSelect from "components/advancedSelectWidget/AdvancedSingleSelect"
+import { customFieldsJSONString } from "components/CustomFields"
+import LocationField from "components/MergeField"
+import Messages from "components/Messages"
+import {
+  CUSTOM_FIELD_TYPE_DEFAULTS,
+  DEFAULT_CUSTOM_FIELDS_PARENT,
+  MODEL_TO_OBJECT_TYPE
+} from "components/Model"
+import {
+  jumpToTop,
+  mapPageDispatchersToProps,
+  PageDispatchersPropType,
+  useBoilerplate
+} from "components/Page"
+import useMergeObjects, {
+  getActionButton,
+  getActivationButton,
+  getClearButton,
+  getInfoButton,
+  getLeafletMap,
+  selectAllFields,
+  setAMergedField,
+  setMergeable
+} from "mergeUtils"
+import { Location } from "models"
+import PropTypes from "prop-types"
+import React, { useState } from "react"
+import { Col, FormGroup, Grid, Row } from "react-bootstrap"
+import { connect } from "react-redux"
+import { useHistory } from "react-router-dom"
+import LOCATIONS_ICON from "resources/locations.png"
+import Settings from "settings"
+import utils from "utils"
+import { convertLatLngToMGRS } from "../../geoUtils"
+import { areAllSet } from "../../mergeUtils"
+import ApprovalSteps from "../locations/ApprovalSteps"
+import GeoLocation from "../locations/GeoLocation"
```

Comment:
`GeoLocation` should probably be moved to `components` now that it is used in multiple places.